config/phenomenal_tools2container.yaml

```diff
@@ -36,14 +36,14 @@ assignment:
   docker_repo_override: container-registry.phenomenal-h2020.eu
   docker_owner_override: phnmnl
   docker_image_override: escher-fluxomics
-  docker_tag_override: v1.6.0-beta.4_cv1.0.9
+  docker_tag_override: dev_v1.6.0-beta.4_cv1.1.19 
```

Comment:
hi @cfoguet.  You should be tagging a release version of the tool, not dev.  Here's a link to the procedure: [link](https://github.com/phnmnl/phenomenal-h2020/wiki/Tool-container-release-process#making-the-release)

Reply:
I have just released escher-fluxomics according to what we have discussed with @cfoguet here: https://github.com/phnmnl/container-escher-fluxomics/pull/1

According to that at least, release tag for this would be `v1.6.0-beta.4_cv1.1.20`. Please use that tag on this pull request @cfoguet.

Reply:
This is essentially the same as `dev_v1.6.0-beta.4_cv1.1.19` that you were using, so it should work fine (but please test locally @cfoguet).

Maybe you want to update `fluxomics-viz.xml` with the one on the container-escher-fluxomics though (changes are mostly documentation wise). However, since the wrapper changes from 0.1 to 1.0, you might want to re-import the workflow files that use that tool. If this is too much hassle for the release, then don't change the fluxomics-viz.xml Galaxy tool wrapper.